src/main/java/kineticrevolution/tileEntities/TileDuster.java

```diff
@@ -50,20 +54,21 @@ public void onFallenUpon(Entity entity, float distance) {
 		IDusterRecipe recipe = DusterRecipeManager.getRecipe(worldObj, xCoord, yCoord - 1, zCoord);
 		if (recipe == null)
 			return;
-		meta++;
-		if (meta >= 4) {
+		progress++;
+		if (progress >= maxProgress) {
+			progress = 0;
 			spawnParticles(30);
 			targetHeight = 1.0;
-			worldObj.setBlock(xCoord, yCoord - 1, zCoord, BlockLoader.duster, 0, 2);
+			worldObj.setBlock(xCoord, yCoord - 1, zCoord, BlockLoader.duster, 0, 3);
 			NBTTagCompound tag = new NBTTagCompound();
+			yCoord--;
 			writeToNBT(tag);
-			tag.setInteger("y", yCoord - 1);
+			yCoord++;
```

Comment:
also kind of not needed i think to reset the y coord, the entity is getting removed anyways